src/utils/manager.py

```diff
@@ -223,12 +268,23 @@ def umount(mountpoint: Union[str, os.PathLike], force: bool = False) -> None:
     Raises:
         Error: Raised if NFS share umount operation fails.
     """
-    cmd = ["umount"]
+    cmd = ["service", "autofs"]
     if force:
-        cmd.append("--force")
-    cmd.append(mountpoint)
+        cmd.append("restart")
+    else:
+        cmd.append("reload")
```

Comment:
Since we're now using `systemd` instead of the mount command, we should likely update the function signature to not use `--force` as an available option. Reload and restart effectively have the same functionality, but reload doesn't stop the service.

`--force` was originally for the case where something happened to the NFS server and we need to make an unclean break to unmount the clients from the server.